### PR TITLE
feat: offer to add default urls for quickstart

### PR DESCRIPTION
**Pull request by aherzog-auth0**
### Description

Add prompts to update an application's allowed urls to include localhost:3000 after downloading quickstart. 

Screen Shot 2021-03-15 at 8 25 34 AM

No prompt if the URLs have already been added
Screen Shot 2021-03-15 at 8 15 20 AM

- [x] Add prompts for adding url
- [x] Check if urls already exist before prompting

### References
https://github.com/auth0/auth0-cli/issues/128

### Testing

> Describe how this can be tested by reviewers. Be specific about anything not tested and reasons why. If this library has unit and/or integration testing, tests should be added for new functionality and existing tests should complete without errors.
>
> Please include any manual steps for testing end-to-end or functionality not covered by unit/integration tests.
>
> Also include details of the environment this PR was developed in (language/platform/browser version).

- [ ] This change adds test coverage for new/changed/fixed functionality

### Checklist

- [ ] I have added documentation for new/changed functionality in this PR or in auth0.com/docs
- [ ] All active GitHub checks for tests, formatting, and security are passing
- [ ] The correct base branch is being used, if not `master`

### CODE REVIEW

**Comment by woloski** [comment]
I would make it a single question for the three props

File: internal/cli/quickstarts.go [hunk 1]
```diff
@@ -32,6 +33,15 @@ var (
 	}()
 )
 
+// QuickStart app types and defaults
+const (
+	QSNative    = "native"
```
Comment (jfatta) [inline internal/cli/quickstarts.go:38]: I these could be internal to the cli package.

File: internal/cli/quickstarts.go [hunk 2]
```diff
@@ -123,6 +134,11 @@ func downloadQuickstart(cli *cli) *cobra.Command {
 			}
 
 			cli.renderer.Infof("Quickstart sample sucessfully downloaded at %s", target)
+
+			qsType := quickstartsTypeFor(client.GetAppType())
+			if err := promptDefaultURLs(context.TODO(), cli, client, qsType); err != nil {
```
Comment (jfatta) [inline internal/cli/quickstarts.go:139]: I missed this on the first review, sorry 🙏
```suggestion
			if err := promptDefaultURLs(cmd.Context(), cli, client, qsType); err != nil {
```

**Review by jfatta — APPROVED** [review]
I left a minor suggestion inline, then 📦 :shipit:

**Review by jfatta — APPROVED** [review]

**Pull request merged**